src/components/Chat/ChatItem.tsx

```diff
@@ -10,54 +11,65 @@ import {
   ChildrenComponentProps,
   ContentComponentProps,
   commonPropTypes,
+  customPropTypes,
 } from '../../lib'
 import Slot from '../Slot/Slot'
+import ChatGutter from './ChatGutter'
 
 export interface ChatItemProps
   extends UIComponentProps,
     ChildrenComponentProps,
-    ContentComponentProps<ShorthandValue> {}
+    ContentComponentProps<ShorthandValue> {
+  /** Chat items can have a gutter. */
+  gutter?: ShorthandValue
+
+  /** Indicates whether message belongs to the current user. */
+  gutterPosition?: 'start' | 'end'
+}
 
 /**
  * A chat item represents a single event in a chat.
  */
 class ChatItem extends UIComponent<Extendable<ChatItemProps>, any> {
   static className = 'ui-chat__item'
-
   static create: Function
-
   static displayName = 'ChatItem'
 
   static propTypes = {
     ...commonPropTypes.createCommon({
       content: 'shorthand',
     }),
+    gutter: customPropTypes.itemShorthand,
+    gutterPosition: PropTypes.oneOf(['start', 'end']),
   }
 
   static defaultProps = {
     as: 'li',
+    gutterPosition: 'end',
```

Comment:
I would expect that gutterPosition: 'start' would be the default one, as it is more often used. Any particular reason why you choose end?

Reply:
no reason, changed to `end`